Extensions/Ansible/Src/Tasks/Ansible/ansibleCommandLineInterface.ts

```diff
@@ -4,6 +4,7 @@ import * as ansibleUtils from './ansibleUtils';
 import { RemoteCommandOptions } from './ansibleUtils'
 import { ansibleTaskParameters } from './ansibleTaskParameters';
 
+var guid = require('guid-typescript');

```

Comment:
Can you instead use [uuid](https://www.npmjs.com/package/uuid) instead. That is the one we use in our tasks

Reply:
@sachinma converted to uuid